docs/licensing/repository-licensing.md

```diff
@@ -0,0 +1,54 @@
+# OpenRewrite licensing by repository
+
+This table designates the licensing type for files in each OpenRewrite repository.
+
+| Repository                   | License Type               |
+|------------------------------|----------------------------|
+| rewrite                      | Apache License Version 2.0 |
```

Comment:
- https://www.apache.org/licenses/LICENSE-2.0
- https://docs.moderne.io/administrator-documentation/licensing/moderne-source-available-license
- https://docs.moderne.io/administrator-documentation/licensing/overview

Reply:
Some of these repositories will not be public so a link won't go anywhere. I don't see much benefit in linking to these repos from this page. 

For the license type - I'll add a reference to the licenses to the doc rather than linking from each row.